misc/net.minetest.minetest.metainfo.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>net.minetest.minetest</id>
+
+    <name>Minetest</name>
+    <summary>Multiplayer infinite-world block sandbox game</summary>
+    <summary xml:lang="de">Mehrspieler-Sandkastenspiel mit unendlichen Blockwelten</summary>
+    <summary xml:lang="es">Juego sandbox multijugador con mundos infinitos</summary>
+    <summary xml:lang="fr">Jeu multijoueur bac à sable avec des mondes infinis</summary>
+    <summary xml:lang="ja">マルチプレイに対応した、無限の世界のブロック型サンドボックスゲームです</summary>
+    <summary xml:lang="ru">Игра-песочница с безграничным миром, состоящим из блоков</summary>
+    <summary xml:lang="tr">Tek-Çok oyuncuyla küplerden sonsuz dünyalar inşa et</summary>
```

Comment:
These descriptions describe Minetest as a game, which is not what Minetest is today. It's an engine, or even more correctly a platform. If you're already at it, it would make sense to write a new description.

Recently I also replaced the description on the Google Play Store.

Reply:
Done!